samples/openthread/cli/CMakeLists.txt

```
@@ -5,11 +5,7 @@
 #
 cmake_minimum_required(VERSION 3.13.1)
 
-set(NRF_SUPPORTED_BOARDS
-	nrf52840dk_nrf52840
-	nrf52833dk_nrf52833
-)
-
+list(APPEND OVERLAY_CONFIG ${ZEPHYR_NRF_MODULE_DIR}/samples/openthread/common/overlay-ot-defaults.conf)
```

Sorry on this.
This actually has to be reverted back to 
```suggestion
list(APPEND OVERLAY_CONFIG ../common/overlay-ot-defaults.conf)
```
because `${ZEPHYR_NRF_MODULE_DIR}` is not defined until `find_package(Zephyr)` has executed.